inc/hubspot-api.php

```diff
@@ -552,4 +552,60 @@ private function createDeal($deal)
 
         return $id;
     }
+
+    /**
+     * Upload files on hubspot in folder
+     * 
+     * @param array $files An array of all files in the form
+     * 
+     * @return array The array having paths of all uploaded files on Hubspot
+     */
+    public function hsFileUpload($files)
+    {
+        if($this->folder != null) {
+            $hsFilesPath = array();
+            foreach ($files as $filesIndex => $fileValue) 

```

Comment:
should be foreach () {

And there is a logic error in here
Please correct me if im wrong but
`break;` will exit out of the loop completely (if you still have files to go then they will not be uploaded)

you will likely want to use 
`continue;` but I believe none of these are needed as you are able to just control the flow using the `if` statement